docs/source/tutorials/token_classification/tutorial-p2.ipynb

```diff
@@ -0,0 +1,522 @@
+      "-----------------------------\n",
+      "labeled as ORG, but predicted as O 7 times\n",
+      "labeled as O, but predicted as ORG 5 times\n",
+      "labeled as O, but predicted as LOC 3 times\n",
+      "labeled as MISC, but predicted as ORG 1 times\n",
+      "\n",
+      "'Philadelphia' is mislabeled 15 times\n",
+      "-----------------------------\n",
+      "labeled as ORG, but predicted as LOC 14 times\n",
+      "labeled as LOC, but predicted as ORG 1 times\n",
+      "\n",
+      "'Usda' is mislabeled 13 times\n",
+      "-----------------------------\n",
+      "labeled as ORG, but predicted as LOC 7 times\n",
+      "labeled as ORG, but predicted as PER 5 times\n",
+      "labeled as ORG, but predicted as MISC 1 times\n",
+      "\n",
+      "'York' is mislabeled 12 times\n",
+      "-----------------------------\n",
+      "labeled as ORG, but predicted as LOC 11 times\n",
+      "labeled as LOC, but predicted as ORG 1 times\n",
+      "\n"
+     ]
+    }
+   ],
+   "source": [
+    "entities = ['O', 'MISC', 'PER', 'ORG', 'LOC'] \n",
+    "common_token_issues(issues, given_words, labels, pred_probs, entities, exclude=[(0, 1), (1, 0)]) "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "a35ef843",
+   "metadata": {},
+   "source": [
+    "## 5. Find issue sentences with particular word \n",
+    "\n",
+    "Call `search_token` to examine the token label issues of a specific token. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 10,
+   "id": "c8f4e163",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Sentence 471: Soccer - Keane Signs Four-year Contract With Manchester \u001b[31mUnited\u001b[0m.\n",
+      "Given label: LOC, predicted label: ORG\n",
+      "\n",
+      "Sentence 15658: \u001b[31mUnited\u001b[0m Nations 1996-08-29\n",
+      "Given label: ORG, predicted label: LOC\n",
+      "\n",
+      "Sentence 19072: The Humane Society of the \u001b[31mUnited\u001b[0m States estimates that between 500,000 and one million bites are delivered by dogs each year, more than half of which are suffered by children.\n",
+      "Given label: LOC, predicted label: ORG\n",
+      "\n",
+      "Sentence 19104: \u001b[31mUnited\u001b[0m Nations 1996-12-06\n",
+      "Given label: ORG, predicted label: LOC\n",
+      "\n",
+      "Sentence 19879: 1. \u001b[31mUnited\u001b[0m States Iii (Brian Shimer, Randy Jones) one\n",
+      "Given label: ORG, predicted label: LOC\n",
+      "\n",
+      "Sentence 19910: His father Clarence Woolmer represented \u001b[31mUnited\u001b[0m Province, now renamed Uttar Pradesh, in India's Ranji Trophy national championship and captained the state during 1949.\n",
+      "Given label: LOC, predicted label: ORG\n",
+      "\n"
+     ]
+    }
+   ],
+   "source": [
+    "token = 'United' \n",
+    "_ = search_token(token, issues, labels, pred_probs, given_words, sentences, entities) "
```

Comment:
Let's call this method: `find_issues_with_token` instead of `search_token`

Reply:
Let's also have this method return a dict or dataframe of all sentences with issues in this particular token